kubernetes.yml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+    name: comrade-colonel-bot-pv
+    labels:
+        type: local
+spec:
+    storageClassName: manual
+    capacity:
+        storage: 50Mi
+    accessModes:
+        - ReadWriteOnce
+        - ReadWriteMany
+    hostPath:
+        path: /var/lib/comrade-colonel-bot
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+    name: comrade-colonel-bot-pv-claim
+spec:
+    storageClassName: manual
+    accessModes:
+        - ReadWriteOnce
+        - ReadWriteMany
+    resources:
+        requests:
+            storage: 50Mi
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+    name: comrade-colonel-bot
+spec:
+    replicas: 1
+    template:
+        metadata:
+            labels:
+                app: comrade-colonel-bot
+        spec:
+            containers:
+            -   name: comrade-colonel-bot
+                image: quasiyoke/comrade-colonel-bot:latest
+                env:
+                -   name: RUST_LOG
+                    value: comrade_colonel_bot=info
+                -   name: STORAGE_PATH
+                    value: /var/lib/comrade-colonel-bot/db.sqlite3
+                -   name: TELEGRAM_BOT_TOKEN
```

Comment:
To be honest, I don't like 4-spaces indenting for yaml. Kubernetes forces me to format arrays this fancy way and reformats it to two spaces inside (`kubectl edit deploy/comrade-colonel-bot`)

Reply:
So... what is the problem? Use two spaces and don't worry about it.

Reply:
I decided to leave it as is.